### PR TITLE
 beta drop update note

### PR DESCRIPTION
**Pull request by kalexand-rh**

### CODE REVIEW

File: updating/updating-cluster.adoc [hunk 1]
```diff
@@ -7,6 +7,18 @@ toc::[]
...
 
+[IMPORTANT]
+====
+During the {product-title} beta period, update functionality is limited:
+
+* You cannot update between different beta drops, such as from OpenShift 4 Beta
+Drop 2 to OpenShift 4 Beta Drop 3.
+* The ability to update from a beta drop of {product-title} to a full version of
+is not guaranteed.
+* After you install OpenShift 4 Beta Drop 3, you can update your to the
```
Comment (vikram-redhat) [inline updating/updating-cluster.adoc]: typo, "update your ...."

**Comment by kalexand-rh** [comment]
@abhinavdahiya, does this note sound right for the beta 3 upgrade restrictions?

File: updating/updating-cluster.adoc [hunk 2]
```diff
@@ -7,6 +7,18 @@ toc::[]
...
+[IMPORTANT]
+====
+During the {product-title} beta period, update functionality is limited:
+
+* You cannot update between different beta drops, such as from OpenShift 4 Beta
+Drop 2 to OpenShift 4 Beta Drop 3.
+* The ability to update from a beta drop of {product-title} to a full version of
+is not guaranteed.
+* After you install OpenShift 4 Beta Drop 3, you can update your cluster to the
+available nightly builds.
```
Comment (abhinavdahiya) [inline updating/updating-cluster.adoc]: `available nightly builds` ?
@sdodson do you know if it is any nightly builds after beta3 for some specific builds ?
also how many night builds all / first x / just next one ?

Comment (sdodson) [reply updating/updating-cluster.adoc]: I believe it will be a set of specifically tested builds.

Comment (sdodson) [reply updating/updating-cluster.adoc]: From the customer's perspective they wouldn't know that, they'll only see the updates we want them to.

Comment (kalexand-rh) [reply updating/updating-cluster.adoc]: Right. I though I heard that it was a set of three validated daily builds, so from a customer's perspective, it'll look like nightly builds. I'm happy to change the text if you think something else would be less confusing for both internal and external audiences.

Comment (sdodson) [reply updating/updating-cluster.adoc]: I think it's fine as written.

**Comment by kalexand-rh** [comment]
@openshift/team-documentation, PTAL?

File: updating/updating-cluster.adoc [hunk 3]
```diff
@@ -7,6 +7,18 @@ toc::[]
 
 You can update, or upgrade, an {product-title} cluster.
 
+[IMPORTANT]
+====
+During the {product-title} beta period, update functionality is limited:
```
Comment (bergerhoffer) [inline updating/updating-cluster.adoc]: * This mentions beta without a version. Should it say `During the {product-title} 4 beta period` ?
* Consider leading in to the bullets like "update functionality has the following limitations" or "update functionality is limited in the following ways"

File: updating/updating-cluster.adoc [hunk 4]
```diff
@@ -7,6 +7,18 @@ toc::[]
...
+
+* You cannot update between different beta drops, such as from OpenShift 4 Beta
+Drop 2 to OpenShift 4 Beta Drop 3.
+* The ability to update from a beta drop of {product-title} to a full version of
```
Comment (bergerhoffer) [inline updating/updating-cluster.adoc]: This drops off. I assume should either be "a full version of {product-title}" or just remove the "of".

File: updating/updating-cluster.adoc (hunk 2 above)
Comment (bergerhoffer) [inline updating/updating-cluster.adoc]: "cluster" is singular, but "builds" is plural. Suggest changing it to "you can update your cluster to one of the
available nightly builds."

**Comment by kalexand-rh** [comment]
Thanks @bergerhoffer!

**Pull request merged**